src/UI Component/Adminpanel/Activities/currentloan.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+
+const CurrentLoan = () => {
+  // Assuming a random data for this    
+  const loanData = [
+    { id: 1, amount: 5000, repaid: true },
+    { id: 2, amount: 8000, repaid: false },
+    { id: 3, amount: 10000, repaid: true },
+    // Add more entries as needed
+  ];
+
+  // State for filtering
+  const [showRepaid, setShowRepaid] = useState(true);
+  const [showUnpaid, setShowUnpaid] = useState(true);
+
+  // Function to filter loans based on repaid status
+  const filterLoans = () => {
+    return loanData.filter((loan) => (showRepaid && loan.repaid) || (showUnpaid && !loan.repaid));
```

Comment:
Please explain what this should do

Reply:
Also, i don't think should be in a function, it can be a constant and be reused on LN 57, 64

Reply:
This line is creating a new array called filterloans by filtering the loanData array based on certain conditions. just to say that filterloans will contain only the loans that match the conditions specified by the showRepaid and showUnpaid states.